### DISCUSSION TITLE
Help!! || Proxy configured & still having issues when docker-compose pulling netbox || Help!!

### DISCUSSION THREAD

**Discussion by jxavier527** [netbox-community/netbox-docker#634] [Getting Help] [answered]
netbox_1               | 🧬 loaded config '/etc/netbox/config/configuration.py'
netbox_1               | 🧬 loaded config '/etc/netbox/config/extra.py'
netbox_1               | 🧬 loaded config '/etc/netbox/config/logging.py'
netbox_1               | 🧬 loaded config '/etc/netbox/config/plugins.py'
netbox_1               | ⚙️ Applying configuration from /etc/unit/nginx-unit.json
netbox_1               | ✅ Unit configuration loaded successfully
netbox_1               | 2021/10/27 15:06:58 [notice] 7#7 process 14 exited with code 0
netbox-housekeeping_1  |        Request error: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/netbox-community/netbox/releases (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2c0a351760>: Failed to establish a new connection: [Errno 110] Operation timed out'))
netbox-housekeeping_1  | Finished.

**Comment by tobiasge** [comment, marked as answer]
From the logs it seems like your Netbox is started correctly. The error in the last line is from the "housekeeping" container, where Netbox runs scheduled tasks. The error you see shows that the update check for Netbox failed with a connection timeout. You can ignore this error.

**Reply by jxavier527** [reply]
thank you!! It has been ignored and the netbox is up :)
